packages/compendium-cdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@compendium-app/compendium-cdk",
-  "version": "0.3.2",
+  "name": "@zatkoa/compendium-cdk",
+  "version": "0.3.3",
```

Comment:
Can we add a info about release process to readme? What should be done? Also didn't we consider using semantic release so we don't have to increase versions manually?

Reply:
As there are multiple packages to publish, it would be not easy to set up semantic-release or run release on the tag creation. If the current approach turns out to be a problem in the future, we can try to implement it.

For now, I will just add the release process to the readme.